src/actions/getDomainMetadata.ts

```diff
@@ -0,0 +1,21 @@
+import { Registrar } from "../contracts/types";
+import { DomainMetadata } from "../types"
+import { makeApiCall } from "../api/actions/helpers";
+
+export const getDomainMetadata = async (domainId: string, registrar: Registrar) => {
```

Comment:
Nit: If you refactored this method to instead take `metadataUri` as the parameter it would be a bit more flexible.